root/etc/e-smith/events/actions/nethserver-roundcubemail-restore-rh-mariadb105-database

```diff
@@ -24,7 +24,7 @@ set -e
 # keep track of the last executed command
 trap 'last_command=$current_command; current_command=$BASH_COMMAND' DEBUG
 # echo an error message before exiting
-trap 'echo "Error while restoring roundcubemail database to SCL mariadb105: \"${last_command}\" command filed with exit code $?."' EXIT
+trap 'echo "Roundcubemail DB restoration : \"${last_command}\" command filed with exit code $?."' EXIT
```

Comment:
It's working but it's maybe better to use the following to really avoid any possible misinterpretation(filed? failed?):

`trap 'echo "Roundcubemail DB restoration : \"${last_command}\" command returned exit code $?."' EXIT`

Reply:
I agree, let's go with @mrmarkuz wording